.github/README.md

```diff
@@ -0,0 +1,51 @@
+<h1 align="center">
+    <img src="./readme_assets/logo.png" alt="Logo" width="125" height="125"></br>
+  Out of Lockdown
+</h1>
+
+<div align="center">
+  View vaccination rates for each District Health Board (DHB) in New Zealand and track daily progress towards the 90% goal, enabling Kiwis to get out of lockdown.
+  <br />
+  <br />
+  <a href="https://github.com/theramis/aucvid/issues/new?assignees=&labels=bug&template=bug_report.md&title=%F0%9F%90%9B%20Bug:%20">Report a Bug</a>
+  ·
+  <a href="https://github.com/theramis/aucvid/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=%F0%9F%92%A1%20Feature:%20">Request a Feature</a>
+</div>
+<div align="center">
+<br />
+
+[![license](https://img.shields.io/github/license/theramis/aucvid.svg?style=for-the-badge)](../LICENSE)
+![PRs welcome](https://img.shields.io/badge/PRs-welcome-ff69b4.svg?style=for-the-badge)
```

Comment:
Is this something we are doing? I feel like we are we may need to put little bit more detail on getting it up and running e.g. certain requirements currently to have github auth token (might be it?)

Reply:
yea fair, I may or may not have copy pasted this readme from somewhere else 😂 

Will remove

Reply:
Sweet ta!